config/currency_normalization_map.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ARN": "ARS",
+  "AZM": "AZN",
+  "CHP": "CLP",
+  "DHS": "AED",
+  "ECD": "XCD",
+  "JAD": "JMD",
+  "JYE": "JPY",
+  "KUD": "KWD",
+  "MZM": "MZN",
+  "NMP": "MXN",
+  "NTD": "TWD",
+  "RDD": "DOP",
+  "RM": "MYR",
+  "RMB": "CNY",
+  "SFR": "CHF",
+  "SID": "SGD",
+  "UKL": "GBP",
+  "WON": "KRW"
+}
```

Comment:
These are currencies where people refer to them often enough with another code while they mean a currency that has another valid code. A common example I've seen at Shopify is that "NTD" is entered while "TWD" is the correct one. By returning a TWD currency instance, the correct currency code will end up in the database/used for ActiveMerchant calls/etc. 

We should take a look at those files again and see if some of those currencies should be placed here. It's a bit of a grey area, e.g. JEP to GBP seems like the wrong thing because even though they're closely related they are different enough to warrant a separate entry.